extensions/EXT_disjoint_timer_query/extension.xml

```diff
@@ -303,23 +313,23 @@ interface EXT_disjoint_timer_query {
         }
 
         // ...at some point in the future, after returning control to the browser and being called again:
-        var disjoint = gl.getParameter(ext.GPU_DISJOINT_EXT);
+        let disjoint = gl.getParameter(ext.GPU_DISJOINT_EXT);
         if (disjoint) {
           // Have to redo all of the measurements.
         } else {
-          var available;
+          let available;
           if (useTimestamps) {
             available = ext.getQueryObjectEXT(endQuery, ext.QUERY_RESULT_AVAILABLE_EXT);
           } else {
             available = ext.getQueryObjectEXT(timeElapsedQuery, ext.QUERY_RESULT_AVAILABLE_EXT);
           }
-
+1
```

Comment:
accidental?

Reply:
Oops, yes - thanks for catching that. Fixed.